### DISCUSSION TITLE
Rethink about persistence of events in database

### DISCUSSION THREAD

**Discussion by laumair** [iotaledger/wallet.rs#922] [Features / Enhancements]
## Description

We persist events in the database and allow consumers to query them. The reason we added this feature was because of a specific consumer requested it. Imo, we need to rethink if this is worth it. This just bloats the database size. 

## Motivation

To make wallet library more scalable / performant. 

## Open questions (optional)

Use this section to ask any questions that are related to the feature.

## Are you planning to do it yourself in a pull request?

No.

**Comment by Thoralf-M** [comment]
I think we shouldn't persist them, users should rely on our functions to get the outputs and transactions and detect deposits based on that. We just need to think about how to design it, so it's easy and clear for everyone
